braid/braid.h

```diff
@@ -731,6 +744,22 @@ braid_SetSpatialRefine(braid_Core         core,   /**< braid_Core (_braid_Core)
                        braid_PtFcnSRefine srefine /**< function pointer to spatial refinement routine */
                        );
 
+/**
+ * Set sync routine with user-defined routine.
+ * Sync gives user access to XBraid and the user's pap at various points
```

Comment:
"user's pap" to "user's app"  @jcc242

Reply:
Whoops! Fixed in cbf1bb00ba48443a77aec25da2d16f2a2007f874